### DISCUSSION TITLE
I don't like the tux model. It looks too much like a kids game

### DISCUSSION THREAD

**Discussion by diablo488** [SuperTux/supertux#2639] [Q&A] [answered]
Is there any way to switch it back to the old models

**Comment by Frostwithasideofsalt** [comment]
ratio 

but like, really? it's a mario clone. what did you expect? do you want it to be super realistic, masculine and gritty? Is playing a game that looks like it's for kids that big of a deal?

**Reply by diablo488** [reply]
i have been playing the game since i was a kid back in 2010 and it feels wierd to see all of the models change into something too much light hearted

**Reply by Frostwithasideofsalt** [reply]
> models change into something too much light hearted

wasn't it also lighthearted before?

**Reply by diablo488** [reply]
not this much

**Reply by tylerandari13** [reply]
> i have been playing the game since i was a kid back in 2010

Ah yes you playing the game definitely justifies your opinion lol

**Reply by diablo488** [reply]
its my opinion if you can't handle leave the discussion

**Reply by Frostwithasideofsalt** [reply]
bro 💀 
"leave if you cant handle my opinion" is like the most meaningless comeback ever

**Reply by tylerandari13** [reply]
bro 💀

**Reply by diablo488** [reply]
its just my opinion. stop fighting

**Reply by UbuntuJackson** [reply]
@diablo488 It looks way more high quality now, I would not complain about that if I were you. Do better, draw better, program better, and suggest better by being more specific.

**Reply by diablo488** [reply]
ok 🙏

**Reply by tylerandari13** [reply]
Whats the point of even making a discussion if you tell people to go away when they disagree? Can we not have a discussion under the discussion?

Edit: Naaahhh blud changed it 😭

**Reply by provigz** [reply]
Not sure what all this is about. I don't agree with their opinion at all, but I think the question is reasonable and you could try to answer it instead of go into "but why don't you like the models", "why do you care so much"...

**Reply by tobbi** [reply]
I agree: People are entitled to their opinion.

**Reply by tylerandari13** [reply]
The initial response was made before it was edited which is probably why it seems so harsh in comparison to what he asked.

**Comment by tylerandari13** [comment]
lol you should see the corrupted forest

**Comment by UbuntuJackson** [comment]
> its just my opinion. stop fighting

I recommend going to the ST soundtrack comment section on YT and say that, recreate that infamous paper mario thousand year door reply chain Moistcritical brought up.

**Comment by provigz** [comment, marked as answer]
It is planned to have a resource pack which reverts textures to their 0.6.3 counterparts. That is the closest you could get to reverting the models.

**Reply by diablo488** [reply]
thanks

**Reply by diablo488** [reply]
this was the type of reply i was expecting instead of "ratio"🙃

**Reply by provigz** [reply]
@diablo488 Keep in mind this may cause visual bugs since old sprites may not support some new actions and etc... Also, some new enemies/objects will not have a modern counterpart, so that would cause visual inconsistencies. But generally, I believe it should do the job.
